config/swagger.yaml

```diff
@@ -37,22 +37,355 @@ paths:
 ## External APIs definitions
 ###
 
-  /top:
+  /blocks/top:
     get:
       tags:
         - external
         - gossip
-      operationId: GetTop
-      description: 'Get the top block header'
+      operationId: GetTopBlock
+      description: 'Get the top block (either key or micro block)'
       produces:
         - application/json
       parameters:
       responses:
         '200':
-          description: successful operation
+          description: 'Successful operation'
           schema:
             $ref: '#/definitions/Top'
       security:
+  /key-blocks/current:
+    get:
+      tags:
+        - external
+        - gossip
```

Comment:
As discussed - we need to address endpoint tags. Tracked in [Revisit swagger tags](https://www.pivotaltracker.com/story/show/158872647)